src/Controls/src/Core/Platform/Windows/Extensions/FormattedStringExtensions.cs

```diff
@@ -25,7 +25,7 @@ public static void UpdateInlines(this TextBlock textBlock, Label label)
 			this TextBlock textBlock,
 			IFontManager fontManager,
 			FormattedString formattedString,
-			double defaultLineHeight = -1d,
+			double defaultLineHeight = 0d, // TODO: should be -1, but too late to change for net6
```

Comment:
Not super essential as this is just the default value and was passed in via the call above. This overload really should have been internal.